collaborative/IOU/vulnerability_assessment.ipynb

```diff
@@ -0,0 +1,192 @@
+    "\n",
+    "Functionality to assess over a gridded area (region) is in the works as well!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "f86bed4b-a58d-4d9b-9a06-adfb5fb89717",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "## select a single custom location\n",
+    "# your_lat = LAT\n",
+    "# your_lon = LON"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c435c50f-dec5-4440-9bd6-e5207a317f92",
+   "metadata": {},
+   "source": [
+    "To import your own custom locations, we recommend putting your csv file in the same folder as this notebook for ease:\n",
+    "1. Drag and drop a csv file into the file tree on the left hand side; or\n",
+    "2. Use the `upload` button (the \"up arrow\" symbol next to the large blue plus symbol above the file tree). \n",
+    "\n",
+    "<span style=\"color:#FF0000\">**Formatting note**</span>: For the code cells below to work, there must be **2 columns labeled `lat` and `lon`**. Functionality to accept different labeling is forthcoming!\n",
+    "\n",
+    "In the cell below, we read the csv file in. We extracted three random locations from the HadISD station list as an example here -- you'll want to replace with your own locations file!"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "dd83d9d2-f753-4a7d-ba0b-cb30dc514b77",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Read in dummy locations from `stations_csv` file\n",
+    "from climakitae.core.paths import stations_csv_path\n",
+    "from climakitae.util.utils import read_csv_file\n",
+    "example_locs = read_csv_file(stations_csv_path, index_col=0)[['LAT_Y', 'LON_X']].rename(columns={'LAT_Y': 'lat', 'LON_X': 'lon'})"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "17a24db2-dc22-41b9-a259-682781a0b0b0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# select a location from the location list\n",
+    "one_loc = example_locs.loc[example_locs.index == 0]\n",
+    "loc_lat = one_loc.lat.values[0]\n",
+    "loc_lon = one_loc.lon.values[0]\n",
+    "print(loc_lat, loc_lon)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "33a18588-8941-416f-b641-5dcd7d342884",
+   "metadata": {},
+   "source": [
+    "### Step 2: Retrieve metric data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ca66a74c",
+   "metadata": {},
+   "source": [
+    "Now, with the above desired points, we can pass in a series of arguments into the below `cava_data` function to retrieve the desired raw data and/or computed data for your CAVA analysis."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "85760d97",
+   "metadata": {},
+   "source": [
+    "Below are examples of some of the available selections for the `variable`, `metric_calc`, `ssp_data`, and `export_method` parameters."
```

Comment:
The parameters could use more explanation for general users, especially export_method.